Packs/Core/ReleaseNotes/3_0_83.md

```diff
@@ -0,0 +1,18 @@
+
+#### Playbooks
+
+##### New: Unprivileged process opened a registry hive
+
+- This playbook is designed to handle the 'Unprivileged process opened a registry hive' alert.
+
+Playbook Stages:
+
+Investigation:
+- The playbook is designed to investigate and respond to unprivileged processes opening registry hives. It examines the unprivileged process that triggered the alert, the command line, and searches for additional suspicious Cortex XSIAM alerts within the same incident in order to determine whether a remediation measure is required.
+
+Remediation:
+- To prevent malicious activity from continuing, the playbook terminates the causality processes that triggered the alert. (Available from Cortex XSOAR 8.8.0).
+
+#### Triggers Recommendations
```

Comment:
@umishkin In last RN's, I didn't see the 'Triggers Recommendations' section; should we keep it or remove it?

Reply:
Uriel said we should keep the 'Triggers Recommendations' section.